lib/main.dart

```diff
@@ -30,7 +30,7 @@ class MyApp extends StatelessWidget {
     return Material(
       child: MaterialApp(
         debugShowCheckedModeBanner: false,
-        title: 'Flutter Demo',
+        title: 'tldr_',
```

Comment:
```suggestion
        title: 'tldr man-pages',
```

Reply:
I did that because that's what's on the [main image](https://github.com/Techno-Disaster/tldr-flutter/raw/master/android/fastlane/metadata/android/en-US/images/featureGraphic.png) on the readme.

Reply:
[Play store](https://play.google.com/store/apps/details?id=wtf.technodisaster.tldr) has `tldr man-pages` though so I changed it to this.

Reply:
Ah, I see, makes sense 👍🏻